src/deluge/gui/ui/keyboard/column_controls/scale_mode.cpp

```diff
@@ -18,6 +18,7 @@
 
 #include "scale_mode.h"
 #include "gui/ui/keyboard/layout/column_controls.h"
+#include <cstdbool>
```

Comment:
this shouldn't be needed, in cpp it just provides the macro  __bool_true_false_are_defined